dev.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 
 IP_ADDRESS=${1:-127.0.0.1}
-DELAY=${2:-10} # interval to wait for dependent docker services to initialize
+DELAY=${2:-60} # interval to wait for dependent docker services to initialize
```

Comment:
If we're going to make delay configurable, we probably need to split up the mysql delay from the solr delay.  Otherwise solr, which usually starts fast, has to wait 60 seconds in all cases :(